src/Pages/ArticleById.jsx

```diff
@@ -24,11 +26,24 @@ export default function ArticleById() {
       });
   }, [article_id]);
 
+  useEffect(() => {
+    setIsLoading(true);
+    getCommentsForArticle(article_id)
+      .then(({ comments }) => {
+        setComments(comments);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setIsError(err.response.data.msg);
+        setIsLoading(false);
+      });
+  }, [article_id]);
+
   if (isLoading) {
     return <p>Loading</p>;
   }
 
-  const dateStamp = () => {
+  const dateStamp = (article) => {
```

Comment:
I've seen that you used this function twice - you could pull one out of the component and export it and import it at the other spot where you're using it, just to reduce duplicate code